sdk/schemaregistry/schema-registry-avro/README.md

```diff
@@ -0,0 +1,130 @@
+# Azure Schema Registry Avro serializer client library for JavaScript
+
+Azure Schema Registry is a cloud-based service that stores schemas for
+serialization. This package provides an Avro serializer that stores only schema
+ID in payload and the full Avro schema in the schema registry.
+
+## Getting started
+
+- Node.js version 8.x.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub]
+- An existing Schema Registry resource
+
+<!--
+TODO: Links not publicly available yet for creating resource
+-->
+
+### Install the `@azure/schema-registry-avro` package
+
+Install the Azure Text Analytics client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/schema-registry-avro
+```
+
+## Key concepts
+
+### SchemaRegistryAvroSerializer
+
+Provides API to serialize to and deserialize from Avro Binary Encoding plus a
+header with schema ID. Uses [SchemaRegistryClient](../schema-registry) to get
+schema IDs from schema content or vice versa.
+
+### Message format
+
+The same format is used by schema registry serializers across Azure SDK languages.
+
+Messages are encoded as follows:
+
+- 4 bytes: Format Indicator
+
+  - Currently always zero to indicate format below.
+
+- 32 bytes: Schema ID
+
+  - UTF-8 hexadecimal representation of GUID.
+  - 32 hex digits, no hyphens.
+  - Same format and byte order as string from Schema Registry service.
+
+- Remaining bytes: Avro payload (in general, format-specific payload)
+
+  - Avro Binary Encoding
+  - NOT Avro Object Container File, which includes the schema and defeats the
+    purpose of this serialzer to move the schema out of the message payload and
+    into the schema registry.
+
+## Examples
+
+### Serialize and deserialize
+
+```javascript
+import { DefaultAzureCredential } from "@azure/identity";
```

Comment:
for README samples written in pure JS we usually show `require` instead of import syntax